src/route/dataset-route.ts

```diff
@@ -101,6 +113,11 @@ apiRoute.get('/:dataset', async (req, res) => {
         res.json({ message: 'Dataset not found... file is null or undefined' });
         return;
     }
+    if (isValidUUID(datasetID) === false) {
```

Comment:
This check is unnecessary if we check for the existence of datasetID as a database object and that query is properly protected against injections.

Reply:
This was implemented to prevent the app dying when asking for a dataset by ID which isn't a valid well formed UUID.  I didn't want to wrap the database request in try catch blocks when I can prevent malformed UUIDs from making it to the query line in first place and reject nice and early.  This will reduce load on the database and means we can kick-out bad requests as early as possible.